### DISCUSSION TITLE
What will be the release date for live chart2 with geared

### DISCUSSION THREAD

**Discussion by anivrutti** [Live-Charts/LiveCharts2#103] [Show and tell]
I want to use live chart geared WPF 
In live chart geared facing issue with zooming in line series
Raised issue @
https://github.com/Live-Charts/Live-Charts/issues/1129

![ZoomIssue](https://user-images.githubusercontent.com/46881696/126675909-8ee49efa-f1a2-409a-a9c6-feddc04c6673.png)

**Comment by beto-rodriguez** [comment]
Right now I can give you early access to the geared package, send a mail to Support@lvcharts.net for more info.

**Reply by anivrutti** [reply]
Is it stable version of live chart?
Cause, I need it for client project and also planning to purchase live chart geared license.

**Reply by anivrutti** [reply]
I have sent email to request live chart early access (from anandenivrutti@gmail.com), please reply back.
